src/util/change-types.ts

```diff
@@ -69,11 +68,30 @@ export const changeStationsTypeInBatch = (
             changeStationType(graph, stnId, newStnType);
         });
 
+/**
+ * Change selected stations' type to newStnType in batch.
+ * @param graph Graph.
+ * @param selected Selected
+ * @param newStnType New station's type.
+ * @returns Nothing.
+ */
+export const changeStationsTypeSelected = (
+    graph: MultiDirectedGraph<NodeAttributes, EdgeAttributes, GraphAttributes>,
+    selected: Set<Id>,
```

Comment:
I believe having `stations: StnId[]` would be better. This will match the method name that only changes the stations and leave the filter thing to the caller.